config/kubernetes/sig-release/teams.yaml

```diff
@@ -244,29 +244,47 @@ teams:
         - alejandrox1 # subproject owner
         - annajung # 1.20 RT Docs Lead
         - bai # 1.20 RT Bug Triage Lead
+        - celestehorgan # 1.20 RT Release Notes Shadow
+        - chris-short # 1.20 RT Communications Shadow 
         - cpanato # Release Manager
+        - divya-mohan0209 # 1.20 RT Communications Shadow
+        - droslean # 1.20 RT Bug Triage Shadow
         - eagleusb # 1.20 RT Docs Shadow
-        - eddiezane # 1.20 RT CI Signal Shadow 
+        - eddiezane # 1.20 RT CI Signal Shadow
+        - erismaster # 1.20 RT Bug Triage Shadow
         - guineveresaenger # subproject owner
         - hasheddan # Release Manager / 1.20 RT Lead Shadow
         - hkamel # 1.20 RT CI Signal Shadow
         - jameslaverack # 1.20 RT Release Notes Lead
         - jeremyrickard # 1.20 RT Lead
-        - jrsapi # 1.20 Communications Lead
+        - jrsapi # 1.20 RT Communications Lead
         - justaugustus # subproject owner
         - kcmartin # 1.20 RT Docs Shadow
-        - kikisdeliveryservice # 1.20 Enhancements Lead
+        - kendallroden # 1.20 RT Enhancements Shadow
+        - kikisdeliveryservice # 1.20 RT Enhancements Lead
+        - kinarashah # 1.20 RT Enhancements Shadow
         - lachie83 # subproject owner / 1.20 Emeritus Adviser
+        - mikejoh # 1.20 RT Enhancements Shadow
+        - mkorbi # 1.20 RT Bug Triage Shadow
+        - monzelmasry # 1.20 RT Bug Triage Shadow
+        - morrislaw # 1.20 RT Enhancements Shadow
         - onlydole # subproject owner
         - palnabarun # 1.20 RT Lead Shadow
         - reylejano-rxm # 1.20 RT Docs Shadow
-        - robertkielty #1.20 RT CI Signal Lead
+        - robertkielty # 1.20 RT CI Signal Lead
+        - salaxander # 1.20 RT Communications Shadow
         - saschagrunert # Release Manager
         - savitharaghunathan # 1.20 RT Lead Shadow
-        - ScrapCodes # 1.20 RT CI Signal Shadow
+        - sayanchowdhury # 1.20 Bug Triage Shadow
```

Comment:
nitpick: others are updated to be prefixed with `1.20 RT`

Reply:
UGH I THOUGHT I GOT THEM ALL. HA thanks @hasheddan